force-app/main/default/classes/BehaviorProfileSendEmail.cls

```diff
@@ -0,0 +1,64 @@
+public class BehaviorProfileSendEmail {
+
+@InvocableMethod(label='Send Behavior Email Alert' description='Behavior Profile Send Email')
+    public static void sendEmail(List<EmailRequest> requestList) {
+		Behavior_Evaluation__c inputCollection = requestList[0].behEvaluations;
```

Comment:
Should add a null check here, just in case the requestList is empty, we don't want it to break.